requirements/travis.in

```diff
@@ -0,0 +1,5 @@
+# Requirements for running tests in Travis
+-c constraints.txt
+
+tox                       # Virtualenv management for tests
+tox-battery               # Makes tox aware of requirements file changes
```

Comment:
We usually also put `codecov` here so we can get a known version instead of getting whatever's latest in `.travis.yml`.